### PR TITLE
chore(Messaggi a valore legale): [IAMVL-32] Add get legal message API

### PR DESCRIPTION
**Pull request by Undermaken**
## Short description
This PR adds the API to retrieve the detail of a legal message starting from its ID

### how to test
- use the dev-server by pointing this branch `IAMVL-29-mvl-attachments`
- [setup the config.ts](https://github.com/pagopa/io-dev-api-server/blob/dc06c4a3e14124040b63639da19b5b3d47af1662/src/config.ts#L75) to serve some legal-messages (🙏🏻  please also consider to review the relative PR https://github.com/pagopa/io-dev-api-server/pull/108)

### CODE REVIEW

File: ts/store/reducers/entities/messages/transformers.ts [hunk 1]
```diff
@@ -101,7 +102,9 @@ const getEUCovidCertificate = ({
  * @param messageFromApi
  */
 export const toUIMessageDetails = (
-  messageFromApi: CreatedMessageWithContentAndAttachments
+  messageFromApi:
+    | CreatedMessageWithContentAndAttachments
+    | CreatedMessageWithContent
```
Comment (pietro909) [inline ts/store/reducers/entities/messages/transformers.ts:107]: Not sure whether we need the union type: `CreatedMessageWithContentAndAttachments` should be the equivalent of a supertype of `CreatedMessageWithContent`? Maybe we can simplify it.

File: ts/features/mvl/saga/index.ts [hunk 2]
```diff
@@ -4,24 +4,25 @@ import { ActionType } from "typesafe-actions";
 import { SessionToken } from "../../../types/SessionToken";
 import { waitBackoffError } from "../../../utils/backoffError";
 import { mvlDetailsLoad } from "../store/actions";
+import { BackendMvlClient } from "../api/backendMvl";
+import { apiUrlPrefix } from "../../../config";
 import { handleGetMvl } from "./networking/handleGetMvlDetails";
 
 /**
  * Handle the MVL Requests
  * TODO: stub entrypoint
```
Comment (pietro909) [inline ts/features/mvl/saga/index.ts:13]: the TODO can probably be removed now

Comment (Undermaken) [reply ts/features/mvl/saga/index.ts]: Sure, thx
removed e8da823b70494bba6a022edb39bb043f98409a3b

File: ts/store/reducers/entities/messages/transformers.ts (hunk 1 above)
Comment (Undermaken) [reply ts/store/reducers/entities/messages/transformers.ts]: super! it works!
22770a206076ccdb31b19fe6803e0e61af383510

**Review by pietro909 — APPROVED** [review]

**Pull request merged**